_docs_integrate/establish-relationships.md

```diff
@@ -75,22 +75,49 @@ How to send a Request via a RelationshipTemplate is explained in detail in the [
 
-### Personalize the RelationshipTemplate
+#### Personalization of a RelationshipTemplate
 
-If the [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) is only for creating a Relationship with a single known [Identity]({% link _docs_integrate/data-model-overview.md %}#identity), you can set that Identity's `address` in the RelationshipTemplate:
+The optional property `forIdentity` of a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) can be added to the [input for creating a RelationshipTemplate]({% link _docs_integrate/establish-relationships.md %}#input-for-creating-a-relationshiptemplate) in order to personalize it.
+If the RelationshipTemplate is only for creating a Relationship with a single known [Identity]({% link _docs_integrate/data-model-overview.md %}#identity), you can set that Identity's `address` in the RelationshipTemplate:
 
 ```jsonc
 {
   // ...
   "content": {
-    // ...
+    // RelationshipTemplateContent or ArbitraryRelationshipTemplateContent
+    ...
   },
-  "forIdentity": "<address that the RelationshipTemplate is intended for>"
+  "forIdentity": "<address of Identity designated for RelationshipTemplate>"
 }
 ```
 
 Only that Identity will be able to continue with establishing a Relationship. This also protects sensitive data of that Identity contained in the RelationshipTemplate from outside access.
 
+#### Password Protection of a RelationshipTemplate
+
+The optional property `passwordProtection` of a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) can be added to the [input for creating a RelationshipTemplate]({% link _docs_integrate/establish-relationships.md %}#input-for-creating-a-relationshiptemplate) to provide password protection.
+This allows the RelationshipTemplate to be protected from unauthorized access.
+Only the Identities that know the password of the RelationshipTemplate will be able to continue with establishing a Relationship.
+If the RelationshipTemplate is to be protected by a password, the corresponding password must be specified within its `passwordProtection.password` property.
+
+```jsonc
+{
+  // ...
+  "content": {
+    // RelationshipTemplateContent or ArbitraryRelationshipTemplateContent
+    ...
+  },
+  "passwordProtection": {
+    "password": "<password of RelationshipTemplate>",
+    "passwordIsPin": <can be set to true if the password is a pin>
```

Comment:
Thanks for the different options. 😉

I don't want to choose option A because this is either the boolean `true` or undefined, but never a string.
Option B means additional effort that we could also do at another time as the formatting does not meet our needs in other places as well.
Option C seems appealing for now. :D